scripts/obtain/dotnet-install.sh

```diff
@@ -790,7 +776,7 @@ temporary_file_template="${TMPDIR:-/tmp}/dotnet.XXXXXXXXX"
 channel="LTS"
 version="Latest"
 install_dir="<auto>"
-architecture="<auto>"
+architecture="x64"
```

Comment:
A nit - should the default be the architecture of the current platform? the 'uname -m' returns a Linux specific name, so we can do:
```sh
CPUName=$(uname -m)
case $CPUName in
i686)
    architecture="x86"
    ;;
x86_64|amd64)
    architecture="x64"
    ;;
armv7l)
    architecture="arm"
    ;;
aarch64)
    architecture="arm64"
    ;;
esac
```